openapi.yaml

```diff
@@ -1914,108 +1914,24 @@ paths:
     delete:
       tags:
         - Event Management
-      summary: Undo ingested events. This replenishes the feature credits and draws down the usages and revenue contribution of the events matching the filter.
+      summary: Undo ingested events.
       description: |
-        #### This API lets you to revert the changes caused by events
+        #### This API lets you to revert the changes caused by events.
 
         - **Usages**: Reduction of all usages associated with this event
         - **Revenue**: Reduction of all revenues associated with this event
         - **Entitlements**: Entitlements(Feature Credits) consumed by this event are granted back to the account.
       operationId: eventCorrection
-      parameters:
-        - $ref: '#/components/parameters/filter_account_id'
-        - $ref: '#/components/parameters/filter_event_id'
       responses:
         '200':
           description: Success response
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/EventsCorrectionResponse'
-              example:
+              examples:
```

Comment:
Check if examples is working fine in mintlify. If not, try changing it to `example`

Reply:
examples is working fine. example is not working at all